src/cmd/event.js

```diff
@@ -0,0 +1,196 @@
+        if (!action) return print_usage();
+        const state = await check_for_event_file();
+        if (!state) {
+            console.log('NSFS event file missing');
+        }
+        const beautify = argv.beautify ? String(argv.beautify) : false;
+        if (action === 'active') {
+            const active_events = await get_active_events();
+            if (active_events.length > 0) {
+                out_put_events(active_events, beautify);
+            } else {
+                console.log('No Active events');
+            }
+        }
+        if (action === 'list') {
+            const time = argv.time ? String(argv.time) : '';
+            const time_limit = Date.parse(time);
+            if (time && !time_limit) {
+                console.error('Error: Date formate is not correct.');
+                print_usage();
+                return;
+            }
+            const severity = argv.severity ? String(argv.severity) : '';
+            if (severity && !SEVERITY.includes(severity)) {
+                console.error('Error: Severity value is not valid.');
+                print_usage();
+                return;
+            }
+            const result = await list_events(time, severity);
+            if (result.length > 0) {
+                out_put_events(result, beautify);
+            } else {
+                console.log('No events found with provided criteria');
+            }
+        }
+    } catch (err) {
+      console.error('NSFS Event command: exit on error', err.stack || err);
+      process.exit(2);
+    }
+}
+
+async function out_put_events(events, beautify) {
+    if (beautify) {
+        console.table(events);
+    } else {
+        console.log(events);
+    }
+}
+
+async function check_for_event_file() {
+    const path = '/var/log/noobaa_events.log';
+    return await fs_utils.file_exists(path);
+}
+
+async function get_active_events() {
+    const events_info = await get_all_events();
+    let active_events = [];
+    events_info.forEach(function(obj, index) {
+        if (obj) {
+            const event = JSON.parse(obj.toString());
+            const message = JSON.parse(event.message.toString());
+            message.message_thread = [];
+            const event_time = new Date(event.timestamp);
+            const time_diff = Date.now() - event_time.getTime();
+            if (time_diff < config.EVENT_ACTIVE_INTERVAL) {
+                message.time = new Date(event_time.getTime());
+                const repeated = structure_repeated_events(message, active_events);
+                if (!repeated || active_events.length === 0) {
+                    message.message_thread.push(message.message);
+                    active_events.push(message);
+                }
+            } else {
+                active_events = sort_events(active_events);
+                return active_events;
+            }
+        }
+    });
+    active_events = sort_events(active_events);
```

Comment:
why do we need to sort the events by severity?

Reply:
@romayalon scale team is expecting the sorted active events based on severity.
```If there are multiple events shown by the command, they will be listed in the order we recommend```
```they be fixed, with the most important event to fix at the top ```

Reply:
got you!